proposals/sar_backscatter.json

```diff
@@ -0,0 +1,78 @@
+{
+    "id": "sar_backscatter",
+    "summary": "Computes backscatter from georeferenced SAR input",
+    "description": "Computes backscatter from georeferenced SAR input.\n\nNote that backscatter computation may require instrument specific metadata that is tightly coupled to the original SAR products. As a result, this process may only work in combination with loading data from specific collections, not with general data cubes.",
+    "categories": [
+        "cubes",
+        "sar"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "description": "Data cube containing raw backscatter input.",
```

Comment:
That's one for the experts, @przell  ?

Reply:
It depends on the input data we are using. I know for EODC it is S1 GRD_IW. I guess for vito it's the same? 
The SNAP process we are currently implementing can only use S1 GRD_IW. For S1 SLC we would need a different processing chain. Would it make sense to specify this explicitly?
S1 GRD (Ground Range Detected) is already projected to the earths surface into square pixels based on WGS84 ellipsoid, but I wouldn't call it georeferenced yet. The pixel values are the amplitude directly taken from the measurement file. 
Here is the official description of esa: https://sentinel.esa.int/web/sentinel/user-guides/sentinel-1-sar/product-types-processing-levels/level-1
To avoid going into details in the process description maybe we could word it like this:
3:  "summary": "Computes backscatter from Sentinel1 Level1 GRD input",
4: "description": "Computes backscatter from Sentinel1 Level1 GRD input. ..."
12: "description": "Data cube containing Sentinel1 Level1 GRD data."

@lforesta @clausmichele @aljacob: Do you think that makes sense?

Reply:
Proposal to keep it vague:
Computes backscatter from SAR input

Reply:
Done.